charts/internal/calico/templates/typha/deployment-calico-typha.yaml

```diff
@@ -17,6 +17,8 @@ metadata:
     scheduler.alpha.kubernetes.io/critical-pod: ''
     cluster-autoscaler.kubernetes.io/safe-to-evict: 'true'
   labels:
+    networking.gardener.cloud/to-public-networks: allowed
+    networking.gardener.cloud/to-apiserver: allowed
```

Comment:
Doesn't typha need to accept traffic from the other components?

Reply:
it does, it needs to accept traffic from calico-node.

Reply:
@neo-liang-sap can we add another policy here not in g/g for calico-node, for example: 
```yaml 
---
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: allow-from-calico-node
spec:
  ingress:
  - from:
    - podSelector:
        matchLabels:
          k8s-app: calico-node
    ports:
    - port: 5473
      protocol: TCP
  podSelector:
    matchLabels:
      k8s-app: calico-typha
  policyTypes:
  - Ingress
```

Reply:
Also to cover the entire use-case, lets add `networking.gardener.cloud/to-dns: allowed` to calico components.

Reply:
thanks, i've pushed the changes